database/bolt/bolt.go

```
@@ -0,0 +1,161 @@
+	// 数据存储的桶名称
+	dataBucketName = `speedtest-ex`
+)
+
+type Storage struct {
+	db *bbolt.DB
+}
+
+var (
+	logDebug = logger.Logw
+	logInfo  = logger.LogInfo
+	logWarn  = logger.LogWarning
```

golangci-lint (1.62.2)

var `logWarn` is unused

(unused)